src/notebook/kernel.ts

```diff
@@ -338,11 +349,21 @@ export class Kernel implements IKernel {
       const cellId = cell_ids[idx];
       const name = names[idx];
       const code = codes[idx];
-      const cellData = new vscode.NotebookCellData(
-        vscode.NotebookCellKind.Code,
-        code,
-        PYTHON_LANGUAGE_ID,
-      );
+      // TODO: Since markdown is special, it should be tagged in payload, opposed to trying to determine here.
+      const markdown = maybeMarkdown(code);
```

Comment:
i didnt like using the native vscode markdown since you can't "run it" which does not make it reactive

Reply:
This matches only "pure" markdown, so the only possible dep is marimo

But this in conjunction with the "run on change", address this a little